resources/views/login.blade.php

```diff
@@ -14,7 +14,11 @@
     @endif
     <style>
         body {
+            @if (config('voyager.multilingual.rtl'))
```

Comment:
Changing to new RTL style
```suggestion
            @if (__('voyager::generic.is_rtl') == 'true')
```